Resources/Prototypes/_DV/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -94,3 +94,96 @@
   - type: Construction
     node: start
     defaultTarget: live
+
+- type: entity
+  parent: [ BaseBorgModule, BaseProviderBorgModule ]
+  id: BorgModuleRadioJammerRecharging # locked behind Syndie exofab
+  description: "A module that comes with a recharging radio jammer."
+  name: radio jammer cyborg module
+  components:
+  - type: Sprite
+    layers:
+    - state: syndicate
+    - state: icon-syndicate
```

Comment:
make an abstract parent BaseBorgModuleSyndie with the sprite to reduce copy paste